sdk/keyvault/keyvault/src/az_keyvault_client.c

```diff
@@ -204,6 +183,11 @@ AZ_NODISCARD az_result az_keyvault_keys_key_get(
   // Add path to request after adding query parameter
   AZ_RETURN_IF_FAILED(az_http_request_builder_append_path(&hrb, key_name));
 
+  // Add version if requested
+  if (!az_span_eq(key_version, AZ_SPAN_NULL)) {
```

Comment:
Can't we just use az_span_is_empty()?
I don't think we need the `AZ_SPAN_NULL` at all. Instead, we can write `(az_span){ 0 }`.

Reply:
In my opinion, we want something that stands out like NULL does. So, AZ_SPAN_NULL is the equivalent of NULL but for spans. It makes the code super readable. (az_span){0} is not readable and not obvious to anyone who has to type it in. I think az_span_is_empty is ok but still not as readable and idiomatic as comparing a variable to (AZ_SPAN_)NULL which has been done in C for many decades now. Comparing against NULL is VERY common and understandable so this is my preference.